zebra-chain/src/transaction/tests/test_vectors.rs

```diff
@@ -1,3 +1,6 @@
+use hex::FromHex;
+use lazy_static::lazy_static;
+
```

Comment:
Suggestion: this test vector file doesn't depend on anything in zebra-chain. So let's move it all to zebra-test::vectors.